conda_forge_tick/update_recipe/version.py

```diff
@@ -557,20 +574,28 @@ def update_version_feedstock_dir(
         )
 
 
-def _update_version_feedstock_dir_local(feedstock_dir, version, hash_type):
-    with open(os.path.join(feedstock_dir, "recipe", "meta.yaml")) as f:
+def _update_version_feedstock_dir_local(
+    feedstock_dir, version, hash_type
+) -> (bool, set):
+    if os.path.join(feedstock_dir, "recipe", "recipe.yaml"):
+        return update_version_v1(feedstock_dir, version, hash_type)
+
+    recipe_path = os.path.join(feedstock_dir, "recipe", "meta.yaml")
+    with open(recipe_path) as f:
         raw_meta_yaml = f.read()
     updated_meta_yaml, errors = update_version(
         raw_meta_yaml, version, hash_type=hash_type
     )
     if updated_meta_yaml is not None:
-        with open(os.path.join(feedstock_dir, "recipe", "meta.yaml"), "w") as f:
+        with open(recipe_path, "w") as f:
             f.write(updated_meta_yaml)
 
     return updated_meta_yaml is not None, errors
 
 
-def _update_version_feedstock_dir_containerized(feedstock_dir, version, hash_type):
+def _update_version_feedstock_dir_containerized(
```

Comment:
This function does not use the recipe_version keywork AFAICT. Should it?

Reply:
Nope! :)